apps/tup-ui/src/App.tsx

```diff
@@ -47,6 +48,8 @@ function App() {
           <Route path="" element={<ProjectDetail />}></Route>
           <Route path=":username" element={<ProjectMember />}></Route>
         </Route>
+        <Route path="system_monitor" element={<Systems />}></Route>
+        <Route path="system_monitor/:tas_name" element={<Systems />} />
```

Comment:
**Note.** This path may change. [I've asked designers.](https://tacc-team.slack.com/archives/CQUA4D5KJ/p1688584311405869)

Reply:
Please update to `/system-status`.

<details><summary>Why?</summary>

- That section is labelled "System Status" in the UI.
- The `system_monitor` is just **one part** of that section.
- The `-` instead of `_` because `_` is the common space separator in website URLs.
- Not just `/system` because H.P. wants to avoid any overlap with high-level root CMS URLs (i.e. `tacc.utexas.edu/system` and `tacc.utexas.edu/portal/system`).*

<sup>* I don't fully understand, but I trust H.P.'s experience with users and user-facing website maintenance, and I know my OCD for consistency influences my personal preference.</sup>

</details>

Reply:
We can definitely do this. However, this will have to be a follow on task as it involves tup-services as well. @jarosenb Can you create a task for this just because I'm not sure if you want to break this down into different tasks based on tup-services vs tup-ui changes?

Reply:
I am surprised that tup-services must change for the Portal app URL to change.

CMS and Portal users won't see tup-services URLs in the browser. How are they linked?

Reply:
Sorry you're right they aren't linked 🙃. I can make this change in this PR without tup-services.